test/js/specs/happy.js

```diff
@@ -409,4 +409,164 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnUtilityBillIcon()
+      poaGuidance.clickOnContinueButton()
+      uploadFileAndClickConfirmButton('national_identity_card.pdf')
+      documentSelector.passportIcon.click()
+      uploadFileAndClickConfirmButton('passport.jpg')
+      uploadFileAndClickConfirmButton('face.jpeg')
+      verificationComplete.verifyUIElements(verificationCompleteCopy)
+    })
+
+    it('should upload Councit Tax Letter and finish flow', async () => {
+      const verificationCompleteCopy = verificationComplete.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnCouncilTaxLetterIcon()
+      poaGuidance.clickOnContinueButton()
+      uploadFileAndClickConfirmButton('national_identity_card.pdf')
+      documentSelector.passportIcon.click()
+      uploadFileAndClickConfirmButton('passport.jpg')
+      uploadFileAndClickConfirmButton('face.jpeg')
+      verificationComplete.verifyUIElements(verificationCompleteCopy)
+    })
+
+    //the test below will fail because of the bug CX-3799, footer hovers the benefits letter cell
+    // it('should upload Benefits Letter and finish flow', async () => {
+    //   const verificationCompleteCopy = verificationComplete.copy()
+    //   goToPoADocumentSelectionScreen()
+    //   poaDocumentSelection.clickOnBenefitsLetterIcon()
+    //   poaGuidance.clickOnContinueButton()
+    //   uploadFileAndClickConfirmButton('national_identity_card.pdf')
+    //   documentSelector.passportIcon.click()
+    //   uploadFileAndClickConfirmButton('passport.jpg')
+    //   uploadFileAndClickConfirmButton('face.jpeg')
+    //   verificationComplete.verifyUIElements(verificationCompleteCopy)
+    // })
+
+    it('should succesfully complete cross device e2e flow with PoA document and selfie upload', async () => {
+      const mobileConnectedCopy = crossDeviceMobileConnected.copy()
+      const uploadsSuccessfulCopy = crossDeviceClientSuccess.copy()
+      const crossDeviceSubmitCopy = crossDeviceSubmit.copy()
+      const verificationCompleteCopy = verificationComplete.copy()
+
+      const copyCrossDeviceLinkAndOpenInNewTab = async () => {
+        const crossDeviceLinkText = crossDeviceLink.copyLinkTextContainer.getText()
+        driver.executeScript("window.open('your url','_blank');")
+        switchBrowserTab(1)
+        driver.get(crossDeviceLinkText)
+      }
+
+      const switchBrowserTab = async (tab) => {
+        const browserWindows = driver.getAllWindowHandles()
+        driver.switchTo().window(browserWindows[tab])
+      }
+      
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnBankIcon()
+      poaGuidance.clickOnContinueButton()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+      copyCrossDeviceLinkAndOpenInNewTab()
+      switchBrowserTab(0)
+      driver.sleep(2000)
+      crossDeviceMobileConnected.verifyUIElements(mobileConnectedCopy)
```

Comment:
I think this is redundant too. The cross device connected screen does not change depending on the flow, so I would avoid adding it again. Same for client success, unless the elements change depending on the flow